plugins/chat/assets/javascripts/discourse/components/chat-message-thread-indicator.hbs

```diff
@@ -3,11 +3,41 @@
   @models={{@message.thread.routeModels}}
   class="chat-message-thread-indicator"
 >
-  <span class="chat-message-thread-indicator__replies-count">
-    {{i18n "chat.thread.replies" count=@message.threadReplyCount}}
-  </span>
-  <span class="chat-message-thread-indicator__view-thread overflow-ellipsis">
-    {{i18n "chat.thread.view_thread"}}{{#if this.threadTitle}}:
-      {{replace-emoji this.threadTitle}}{{/if}}
-  </span>
+  <div class="chat-message-thread-indicator__last-reply-avatar">
+    <ChatUserAvatar
+      @user={{@message.thread.preview.lastReplyUser}}
+      @avatarSize="small"
+    />
+  </div>
+  <div class="chat-message-thread-indicator__body">
+    <div class="chat-message-thread-indicator__last-reply-metadata">
+      <span class="chat-message-thread-indicator__last-reply-username">
+        {{#if this.site.mobileView}}
+          <div class="chat-message-thread-indicator__last-reply-avatar -mobile">
+            <ChatUserAvatar
+              @user={{@message.thread.preview.lastReplyUser}}
+              @avatarSize="small"
+            />
+          </div>
+        {{/if}}
+        {{@message.thread.preview.lastReplyUser.username}}
+      </span>
+      <span>{{i18n "chat.thread.last_reply"}}</span>
+      {{format-date @message.thread.preview.lastReplyCreatedAt leaveAgo="true"}}
+      |
+      <span class="chat-message-thread-indicator__replies-count">
+        {{i18n "chat.thread.replies" count=@message.thread.preview.replyCount}}
+      </span>
+
+    </div>
+    <div
+      class="chat-message-thread-indicator__last-reply-excerpt overflow-ellipsis"
```

Comment:
is overflow-ellipsis different than using `@include ellipsis;` in the css?

Reply:
I have no idea, I've just used it / seen it a few times now. Will try the @include